recipe_scrapers/bergamot.py

```diff
@@ -0,0 +1,84 @@
+# mypy: allow-untyped-defs
+import requests
+
+from ._abstract import HEADERS, AbstractScraper
+from ._utils import url_path_to_dict
+
+
+class Bergamot(AbstractScraper):
+    def __init__(self, url, proxies=None, timeout=None, *args, **kwargs):
+        super().__init__(url=url, proxies=proxies, timeout=timeout, *args, **kwargs)
+
+        url_dict = url_path_to_dict(url)
+        path = url_dict.get("path")
+        recipe_id = path.split("/")[-1]
+
+        data_url = f"https://api.bergamot.app/recipes/shared?r={recipe_id}"
+        response = requests.get(
+            data_url, headers=HEADERS, proxies=proxies, timeout=timeout
+        )
+        self.data = response.json()
+
+    @classmethod
+    def host(cls):
+        return "dashboard.bergamot.app"
+
+    def canonical_url(self):
+        source_url = self.data.get("sourceUrl")
+        return source_url if source_url else self.url
+
+    def author(self):
+        return None
```

Comment:
Let's try to find some basic authorship information when we can, even if it is not perfect/ideal.

When a `source_url` is found, I think we could return the domain name.. not ideal, but better than nothing.  I've asked in the bugreport (#986) whether we know of any example recipes that contain more author info.